### DISCUSSION TITLE
scraping without limitations

### DISCUSSION THREAD

**Discussion by tallevy22** [kevinzg/facebook-scraper#890] [Q&A]
Hi ,
is there a way to scrape without limitation ? 
or scraping a group by keywords?
thank you

**Comment by evildrome** [comment]
If you mean scrape without getting a temporary ban from FB..  the answer is no.

You have to put delays in your code to avoid getting banned.  I have a random delay of 30 - 45 seconds between getting each post.

You cannot scrape selectively. You have to scrape all the data and then filter out the stuff you don't want.
